iconrpcserver/dispatcher/default/node.py

```diff
@@ -55,13 +55,17 @@ class NodeDispatcher:
 
             validate_jsonschema_node(request=req_json)
         except GenericJsonRpcServerError as e:
-            response = ExceptionResponse(e, request_id=req_json.get('id', 0))
+            response = ExceptionResponse(e, id=req_json.get('id', 0), debug=True)
         except Exception as e:
-            response = ExceptionResponse(e, request_id=req_json.get('id', 0))
+            response = ExceptionResponse(e, id=req_json.get('id', 0), debug=True)
         else:
-            response = await methods.dispatch(req_json, context=context)
+            if "params" in req_json:
+                req_json["params"]["context"] = context
+            else:
+                req_json["params"] = {"context": context}
+            response = await async_dispatch(json.dumps(req_json), methods)
```

Comment:
what type is this response?

Reply:
It will be `jsonrpcserver.response.Response`, not the pure return value of dispatch method.

Reply:
Thank you. 
I added `jsonrpcserver.response.DictResponse`.
(Currently all dispatch methods return `dict`, which makes `dispatch` return `DictResponse`)